src/styles/product-card/card.scss

```diff
@@ -1,28 +1,28 @@
 .main {
   padding: 50px 40px;
   display: grid;
-  grid-template-columns: repeat(4, 198px);
+  grid-template-columns: repeat(4, 200px);
   gap: 46px 48px;
 
   @media screen and (min-width: 1200px) {
-    padding-top: 50px;
+    padding: 50px 128px;
```

Comment:
Had to add horizontal padding because without it 1 test can`t be passed...
![image](https://github.com/mate-academy/layout_catalog/assets/162454901/6deed2ca-61d4-43a4-bb69-8739ce74bee9)